packages/sdk/src/controllers/VariableController.ts

```diff
@@ -134,6 +155,13 @@ class DateVariable {
 
     /**
      * @experimental This method sets the locale for a date variable.
+     *
+     * Supported languages are (between brackets is the `locale`):
+     *
+     * Czech (`cs`), Danish (`da`), Dutch (`nl`), English (`en_US`), Finnish (`fi`),
+     * French (`fr`), German (`de`), Italian (`it`), Norwegian (`no`), Polish (`pl`),
+     * Portuguese (`pt_PT`), Spanish (`es_ES`), Swedish (`sv`)
```

Comment:
Not sure if this had value given that Locale is an enum

Reply:
You meant on SDK side? On action-side I assume it gives context to Genie. On SDK maybe it's redundant indeed.

Reply:
I think genie is smart enough to understand the enum but perhaps I am wrong